upt_macports/templates/base.Portfile

```diff
@@ -4,19 +4,23 @@ PortSystem          1.0
 {% block portgroup %}
 {% endblock %}
 
-# Portfile generated/maintained using the Universal Packaging Tool (upt)
+## Portfile generated/maintained using the Universal Packaging Tool (upt)
+## Please check out the commented statements and refer to the guide for more information:
+## https://guide.macports.org/#reference.keywords
 {% block nameversion %}
 {% endblock %}
 
-# Please adapt the lines with a "TODO" statement and check the guide for more information:
-# https://guide.macports.org/#reference.keywords
-
-# categories-append # TODO
+# categories-append
 platforms           darwin
-# TODO: uncomment the following line if no architecture-dependent files are installed, otherwise remove
+## uncomment the following line if no architecture-dependent files are installed, otherwise remove
 # supported_archs     noarch
 license             {{ pkg.licenses }}
-# maintainers         {example.org:julesverne @GitHub-handle} openmaintainer # TODO
+## If you would like to keep maintaining this port in the future as described in
+## https://guide.macports.org/chunked/project.contributing.html then
+## please add your contact infromation (e-mail, github handle)
```

Comment:
@Korusuke, if you correct the typo (and perhaps change the capitalization and add punctuation) here and change the PR from "Draft" to "Ready" (of whatever it's called), I think we it should be good to go! 

```
## please add your contact information (e-mail and GitHub handle), 
```